static/templates/settings/notification_settings.hbs

```diff
@@ -51,10 +51,10 @@
     <div class="desktop_notifications m-10 {{#if for_realm_settings}}org-subsection-parent{{else}}subsection-parent{{/if}}">
 
         <div class="subsection-header inline-block">
-            <h3>{{t "Desktop message notifications" }}</h3>
-            {{> settings_save_discard_widget section_name="desktop-message-settings" show_only_indicator=(not for_realm_settings) }}
+            <h3>{{t "Desktop message notifications" }} </h3>
+            <a href="/help/desktop-notifications" target="_blank"><i class="fa fa-question-circle-o settings-info-icon"></i></a>
+            {{> settings_save_discard_widget section_name="desktop-message-settings" show_only_indicator=(not for_realm_settings)  }}
```

Comment:
Looks like you accidentally added an extra space on this line?

Reply:
Thanks for the review, I will correct the changes.

Reply:
It looks like you removed a space that should be there and still have an extra space where it's not needed. Please review the diff *carefully* before requesting another review.

Reply:
Thanks for the review, I have made the changes you suggested could you please have a look at it.

Reply:
Why is there still a random space change? See old line 55 / new line 56.

![Screen Shot 2022-11-01 at 10 02 58 PM](https://user-images.githubusercontent.com/2090066/199402423-0d767d57-2e8a-4a29-b534-e1e5946ac615.png)

Reply:
All right, got it, and many thanks for the review; the PR has been updated to incorporate your comments, so please check it out, Thanks.